packages/core/entity/FileSelection/test/FileSelection.test.ts

```diff
@@ -674,6 +677,9 @@ describe("FileSelection", () => {
             const fileSet1 = new FileSet();
             const fileSet2 = new FileSet({
                 filters: [new FileFilter("foo", "bar")],
+                fuzzyFilters: [new FuzzyFilter("fuzzyfoo1"), new FuzzyFilter("fuzzyfoo2")],
```

Comment:
in cases like this, I like using string variables, e.g. fuzzyFilterValue1 = "fuzzyfoo1", or just setting the names to something more meaningful, like FuzzyFilter("fuzzyFilterValue1") so the validation is easier to follow.

Reply:
I was being lazy. Will update